llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -704,9 +704,10 @@ class LoopAccessInfo {
   const PredicatedScalarEvolution &getPSE() const { return *PSE; }
 
 private:
-  /// Analyze the loop.
-  void analyzeLoop(AAResults *AA, LoopInfo *LI,
-                   const TargetLibraryInfo *TLI, DominatorTree *DT);
+  /// Analyze the loop. Return a boolean indicating whether the memory can be
```

Comment:
Might be simpler to say `Returns true if all memory accesses in the loop can be vectorized.`